app/assets/stylesheets/cieszyn.scss

```diff
@@ -9,9 +9,24 @@ body.cieszyn {
     text-align: justify;
   }
 
-  table {
+  .warranty {
+    table {
+      @extend .table;
+      @extend .table-hover;
+      @extend .table-striped;
+    }
+  }
+
+  table.prices {
     @extend .table;
-    @extend .table-hover;
-    @extend .table-striped;
+    th {
+      background-color: #c3261c;
+      color: white;
+      font-size: 20px;
+    }
+    tr.checked {
```

Comment:
Avoid qualifying class selectors with an element.<br>Selector should have depth of applicability no greater than 2, but was 3